matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/Matrix.kt

```diff
@@ -108,6 +110,11 @@ class Matrix(context: Context, matrixConfiguration: MatrixConfiguration) {
      */
     fun legacySessionImporter() = legacySessionImporter
 
+    /**
+     * Returns the SecureStorageService used to encrypt and decrypt sensitive data.
```

Comment:
Maybe also tell about the storage facility of this service.

Reply:
Maybe the details describing the purpose of this service could be added in the interface file?